File: fizzbuzz.rb
def fizzbuzz(n)
  if n % 3 == 0 && n % 5 == 0
    'FizzBuzz'
  elsif n % 3 == 0
    'Fizz'
  elsif n % 5 == 0
    'Buzz'
  else
    n
  end
end

(1..20).each do |n|
  puts fizzbuzz(n)
end

File: fizzbuzz.r
fizzbuzz <- function(n) {
  if ((n %% 3 == 0) & (n %% 5 == 0)) {
    return("FizzBuzz")
  } else if (n %% 3 == 0) {
    return("Fizz")
  } else if (n %% 5 == 0) {
    return("Buzz")
  } else {
    return(n)
  }
}

for (n in 1:20) {
  print(fizzbuzz(n))
}
